packages/core/src/Account.ts

```diff
@@ -238,11 +238,13 @@ export class Account extends TypedEventEmitter<Events> {
     displayName,
     handle,
     discoveryUrl,
+    hasActiveCertificate,
```

Comment:
why not querying coreCrypto down the line rather than expecting the consumer to give that information?